ui/src/app/layout/topics/topicmetadatatable/topic-metadata-table.component.html

```diff
@@ -8,7 +8,10 @@
         </tr>
         <tr>
             <th scope="row">{{ 'Topic Type' | translate }}</th>
-            <td>{{ topic.topicType === 'DATA' ? 'MASTER DATA' : topic.topicType}}</td>
+            <td>
+                {{topic.topicType === 'DATA' ? ("Master Data" | translate).toUpperCase()
```

Comment:
If using the enum keys as translation keys, can greatly be simplified. No need to use upper case here for display.